ui/apps/dashboard/src/utils/useRestAPIRequest.ts

```diff
@@ -12,9 +12,11 @@ import {
 export function useRestAPIRequest<T>({
   url,
   method,
+  pause = false,
 }: {
   url: string | URL | null;
   method: string;
+  pause: boolean;
```

Comment:
Added this feature similar to URQL to prevent the URL from being hit dynamically. The `url` check on line 28 should have done this, but it might not have updated for some reason. Leaving in for now.